Source/PluginEditor.cpp

```diff
@@ -31,9 +39,23 @@ SoomplerAudioProcessorEditor::SoomplerAudioProcessorEditor (SoomplerAudioProcess
     stopSampleButton.setButtonText(Strings::BUTTON_STOP_SAMPLE_TEXT);
     stopSampleButton.addListener(this);
 
+    volumeKnob.setSliderStyle(Slider::SliderStyle::RotaryHorizontalVerticalDrag);
+    volumeKnob.addListener(this);
+    volumeKnob.setRange(0.0, 1.0, 0.01);
+    volumeKnob.setTextBoxStyle(Slider::NoTextBox, false, 0, 0);
+    volumeKnob.setName("Volume");

```

Comment:
Add to string constant to i18n in future

Reply:
will create issue for next development iteration